CMakeLists.txt

```diff
@@ -95,7 +95,7 @@ SET( SRCS
 if(NOT ${PULL_OPEN62541})  # this is the default behaviour.
 	include_directories(extern/open62541/include)
 	list(APPEND SRCS extern/open62541/src/open62541.c)
-	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99")
+	set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -std=c99 -DUA_ARCHITECTURE_POSIX -Werror -Wall -Wextra -Wpedantic -Wno-static-in-inline -Wno-overlength-strings -Wno-unused-parameter -Wmissing-prototypes -Wstrict-prototypes -Wredundant-decls -Wformat -Wformat-security -Wformat-nonliteral -Wuninitialized -Winit-self -Wcast-qual -Wstrict-overflow -Wnested-externs -Wmultichar -Wundef -Wc++-compat -fno-strict-aliasing -fexceptions -fstack-protector-strong -fstack-clash-protection -Wno-unused-function -Wshadow -Wconversion -fvisibility=hidden")
```

Comment:
not for windows apparently...
```
Command line error D8021: invalid numeric argument '/Wno-literal-suffix'
```
Which of these many options are critical ? I'm thinking could either
* see if there are CMAKE 'standard' equivalents (i.e. that cmake knows how to map into compiler specific options)
* resort to if(WIN32) logic: hence the question about which options are mandatory, so that they, at least, are mapped to visual studio

Reply:
where did this list come from? Open62541 recommends/requires all of these on linux?

Reply:
Those are the default build flags when you clone open62541 1.0, and you build it on Linux.
I trust you can propose proper options on Windows, so as long as I'm not blocked with it, feel free to suggest either solution